chef_master/source/aws_opsworks_chef_automate.rst

```diff
@@ -68,3 +68,205 @@ Enabling the compliance profile storage service in AWS OpsWorks for Chef Automat
+
+To simplify the process, you can run ``sudo -s`` from each of your SSH connections to run commands using ``root`` privileges.
+
+From your Chef Automate server, export the path to the ``knife`` executable to make these commands easier to run.
+
+.. code-block:: bash
+
+   $ export PATH=/opt/opscode/embedded/bin:$PATH
+
+In the steps that follow, you'll need to replace placeholder values with yours. Gather the following information about your environment. The **Placeholder** column lists the placeholder text you'll replace in the steps that follow.
+
++----------------------------------+----------------------------------------------------+---------------------------+
+| Description                      | Example                                            | Placeholder               |
++==================================+====================================================+===========================+
+| Your Chef Automate server's FQDN | ``test-ntex3ai8ej1yhnqu.us-east-1.opsworks-cm.io`` | ``CHEF_AUTOMATE_FQDN``    |
++----------------------------------+----------------------------------------------------+---------------------------+
+| The SSH key you use to connect   | ``id_rsa``                                         | ``CHEF_AUTOMATE_SSH_KEY`` |
+| to your Chef Automate server     |                                                    |                           |
++----------------------------------+----------------------------------------------------+---------------------------+
+| Your build node's IP address     | ``52.23.235.94``                                   | ``BUILD_NODE_IP_ADDRESS`` |
++----------------------------------+----------------------------------------------------+---------------------------+
+| Your build node's user name      | ``ec2-user``                                       | ``BUILD_NODE_USERNAME``   |
++----------------------------------+----------------------------------------------------+---------------------------+
+| The SSH key you use to connect   | ``id_rsa``                                         | ``BUILD_NODE_SSH_KEY``    |
+| to your build node               |                                                    |                           |
++----------------------------------+----------------------------------------------------+---------------------------+
+
+You'll also need to specify the name of an SSL certificate file that matches a special format. This format takes your Chef Automate server's FQDN, replaces the dot ``.`` character with an underscore ``_`` and ends with ``_crt``. For example, if your Chef Automate server's FQDN is:
+
+.. code-block:: bash
+
+   test-ntex3ai8ej1yhnqu.us-east-1.opsworks-cm.io
+
+Then your SSL certificate file name would be:
+
+.. code-block:: bash
+
+   test-ntex3ai8ej1yhnqu_us-east-1_opsworks-cm_io_crt
+
+This certificate file name appears as ``CHEF_AUTOMATE_SSL_CERT`` in the steps that follow.
```

Comment:
[nit] Wouldn't it make sense to include it in the "placeholder table" above, then?